task_8/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+namespace Task_8
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите значение аргумента функции");
+            var x = double.Parse(Console.ReadLine());
+
+            Console.WriteLine($"f({x}) = {MyFunction(x)}");
+
+            Console.ReadKey();
+        }
+
+        static double MyFunction(double x)
+        {
+            if (x < 2)
+                return x - 2;
+            else if (x <= 3)
+                return 3 - x;
+
+            return 0;

```

Comment:
Код не соответствует заданию. Перепутали случаи.
```suggestion
            if (x < 2)
                return x - 2;
            else if (x <= 3)
                return 0;

            return 3 - x;
```
Попробуйте построить график этой функции.